.github/actions/install-oppia-dependencies/action.yml

```
@@ -0,0 +1,24 @@
+name: 'Install Dependencies'
+description: 'Installs wheel and third-party python libs.'
```

```suggestion
description: 'Installs wheel and third-party libs.'
```

(It installs both ts/python deps, right?)

---

done